package.json

```diff
@@ -1,8 +1,23 @@
 {
-  "name": "gatsby-starter-default",
-  "description": "Gatsby default starter",
-  "version": "1.0.0",
-  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
+  "name": "reactiflux.com",
+  "version": "π",
+  "private": true,
+  "license": "MIT",
+  "scripts": {
+    "build": "gatsby build",
+    "deploy": "gatsby build --prefix-links && gh-pages -d public",
+    "start": "gatsby develop",
+    "test": "prettier --check 'src/**/*.{js,md}'",
+    "prettier": "prettier --write 'src/**/*.{js,md}'",
+    "install-if-package-changed": "git diff-tree -r --name-only --no-commit-id ORIG_HEAD HEAD | grep --quiet yarn.lock && yarn install || exit 0"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged",
```

Comment:
pretty-quick is the objectively best way to use prettier and I won't be convinced otherwise